### PR TITLE
[candi] improve bundle detection

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
## Description

1. Improve candi bundle detection to detect centos-based distros.
2. Updated flannel to 0.15.1, due to https://github.com/flannel-io/flannel/issues/1155.
## Changelog entries

Based on https://github.com/deckhouse/deckhouse/pull/1193

```changes
section: candi
type: feature
summary: Improve candi bundle detection to detect centos-based distros
---
section: cni-flannel
type: feature
summary: Bump flannel to 0.15.1.
```

### CODE REVIEW

**Comment by konstantin-axenov** [comment]
@RomanenkoDenys I think you should add information about flannel update to PR description and changelog.

**Comment by RomanenkoDenys** [comment]
> 

Added, thanks.

File: candi/bashible/detect_bundle.sh [hunk 1]
```diff
@@ -32,12 +32,15 @@ if [ -e /etc/os-release ]; then
       >&2 echo "ERROR: Can't determine OS! No ID and VERSION_ID in /etc/os-release."
```
Comment (konstantin-axenov) [inline candi/bashible/detect_bundle.sh:32]: Maybe we should also remove this? Is it possible that `/etc/os-release` doesn't exist?

Comment (RomanenkoDenys) [reply candi/bashible/detect_bundle.sh:32]: I don't know, but i think we can remove this if due to os-release inclusion won't success if file is absent.

**Review by konstantin-axenov — APPROVED** [review]

**Pull request merged**